content/services/hosting/projects.csv

```diff
@@ -67,7 +67,7 @@
 "Harmony Agreements","webapp, mailing-list"
 "HHVM","mirroring, powerpc"
 "Hortonworks Data Platform","powerpc"
-"ICU","powerpc"
+"ICU","build-farm"
```

Comment:
I don't think they're using any of our FB servers, right? This should just be removed I think.

Reply:
In the email they mention that they are using the compile farm stuff, is this false?

Reply:
Hmm, you're right however our ``build-farm`` is referring to the FB servers. Let's go ahead and remove them for now. I prefer we only list projects we directly host, not via other communities (at least for now).